packages/dds/tree/src/simple-tree/api/typesUnsafe.ts

```diff
@@ -51,7 +51,7 @@ export type Unenforced<_DesiredExtendsConstraint> = unknown;
 /**
  * {@link Unenforced} version of `ObjectFromSchemaRecord`.
  * @remarks
- * Do note use this type directly: its only needed in the implementation of generic logic which define recursive schema, not when using recursive schema.
+ * Do not use this type directly: its only needed in the implementation of generic logic which define recursive schema, not when using recursive schema.
```

Comment:
Nit: same for the other comments as well

```suggestion
 * Do not use this type directly: it's only needed in the implementation of generic logic which define recursive schema, not when using recursive schema.
```

Reply:
Good catch! I really gotta stop making that mistake...

Reply:
Fixed.